workshopapps/engineerprofile360.web
# Feat/bac 20 get assessments by organization

#### Task reference
This section contains a link to a corresponding task in **Linear**.
https://linear.app/team-axle-engineer-360-project/issue/BAC-20/getassessmentsbyorgid
#### Summary
This section contains a brief description of **_what was accomplished_** within the current PR.
Create a method to get all assesment for an organization using the organization id
#### Implementation details
This section describes the **_technical aspects_** of the solution, implemented withing the current PR, that are worth mentioning to the reviewers.
Created a getAssByOrgId() method in the AssessmentController and a GET /api/assessment/get/{org_id} route

#### How to test
This section contains a checklist, describing basic scenarios that would help verify that the submitted code works correctly.
Send a get request to api/assessment/get/{org_id} and if successfull will return a 
    {
        "error": true,
        "code": 200,
        "message": "some message here",
        "data": assessment{ l }
    }
#### References 
Any additional information (screenshots, video, diagrams) that would help understand and review the PR.
